cql3/Cql.g

```diff
@@ -1545,6 +1554,20 @@ serviceLevelOrRoleName returns [sstring name]
 | QMARK {add_recognition_error("Bind variables cannot be used for service levels or role names");}
 ;
 
+serviceLevelNameExpr returns [uexpression expr]
+@init{sstring name;}
+: t=IDENT              { name = sstring($t.text);
+                         std::transform(name.begin(), name.end(), name.begin(), ::tolower);
+                         $expr = untyped_constant{untyped_constant::string, std::move(name)}; }
+| t=QUOTED_NAME        { name = sstring($t.text);
+                         $expr = untyped_constant{untyped_constant::string, std::move(name)}; }
+| k=unreserved_keyword { name = k;
+                         std::transform(name.begin(), name.end(), name.begin(), ::tolower);
+                         $expr = untyped_constant{untyped_constant::string, std::move(name)}; }
+| c=constant           { $expr = std::move(c); }
+| e=marker             { $expr = std::move(e); }
+;
```

Comment:
I thought that we discussed whether we should add support for the bind marker or not, and we decided not to do it because we don't support it in other places. Were there something that made you change your mind?

Reply:
Main reason we decided to not support bind marker was I didn't know how to parse SL name as expression(not as string). But after our chat I looked a little bit and found out how, so I've added support for it.

But after comments in the PR, I'll fall back to not supporting it.